analytics/proguard-rules-lib.txt

```diff
@@ -0,0 +1,3 @@
+-keepclassmembers class com.segment.analytics.** {
```

Comment:
i would feel more comfortable if we restricted this a bit. maybe something like
```suggestion
-keepclassmembers class com.segment.analytics.** extends com.segment.analytics.ValueMap {
```
@Sloy would u agree with that?

Reply:
Absolutely, I didn't think of that 👍